grpc/weaviate.proto

```diff
@@ -140,6 +150,7 @@ message AdditionalProperties {
   bool score = 7;
   // protolint:disable:next FIELD_NAMES_LOWER_SNAKE_CASE
   bool explainScore = 8;
+  bool is_consistent = 9;
```

Comment:
why only `TextArray` has this `is_consitent` field and no other `Array types` have it?